### PR TITLE
fix(collision): typo

### PR DESCRIPTION
**Pull request by zaunist**

### CODE REVIEW

**Comment by zaunist** [comment]
@spydon Please take a look, thanks.

File: docs/1.3.0/_sources/flame/collision_detection.md.txt [hunk 1]
```diff
@@ -38,7 +38,7 @@ class MyGame extends FlameGame with HasCollisionDetection {
 }
 ```
 
-Now when you add `ShapeHitbox`s to components that are then added to the game, they will
+Now when you add `ShapeHitbox` to components that are then added to the game, they will
```
Comment (spydon) [inline docs/1.3.0/_sources/flame/collision_detection.md.txt:41]:
```suggestion
Now when you add `ShapeHitbox`es to components that are then added to the game, they will
```

Comment (spydon) [reply docs/1.3.0/_sources/flame/collision_detection.md.txt:41]: It needs to be plural for the rest of the sentence to work.

**Comment by spydon** [comment]
Thanks for the fix, this is the wrong repository though, these are just generated files coming from that flame monorepo:
https://github.com/flame-engine/flame/tree/main/doc

**Comment by zaunist** [comment]
Thanks for your reply.

**Pull request closed without merging**